test/integration/models/test_linode.py

```diff
@@ -138,6 +138,10 @@ def get_status(linode: Instance, status: str):
     return linode.status == status
 
 
+def get_type(linode: Instance, type: str):
```

Comment:
Just to make things a bit clearer, could this be renamed to something like `instance_type_condition`?